build.gradle

```diff
@@ -16,45 +16,52 @@
  */
 
 plugins {
-    id 'net.wooga.plugins' version '1.4.0'
+    id 'net.wooga.plugins' version '3.0.0-rc.2'
 }
 
 group 'net.wooga.gradle'
 description = 'Node release plugin for Gradle.'
 
 repositories {
-    jcenter()
     mavenCentral()
     maven {
         url "https://plugins.gradle.org/m2/"
     }
 }
 
 dependencies {
-    testCompile('org.jfrog.artifactory.client:artifactory-java-client-services:2.+') {
+    testImplementation('org.jfrog.artifactory.client:artifactory-java-client-services:2.9.2') {
         exclude module: 'logback-classic'
     }
-    testCompile 'org.kohsuke:github-api:latest.release'
-    testCompile 'com.github.stefanbirkner:system-rules:1.18.0'
+    testImplementation 'org.kohsuke:github-api:1.301'
```

Comment:
Here we should double check. I would like to have a open version range but this library did breaking changes in the past without warning.

Reply:
I guess pin just to the minor?